tripal/tests/modules/tripal_test/tripal_test.info.yml

```diff
@@ -0,0 +1,6 @@
+type: module
+name: Tripal Test Module
+core_version_requirement: ^8 || ^9 || ^10
```

Comment:
We don't support 8 or 9 any more, should this be changed?

Reply:
Yes, it should
```suggestion
core_version_requirement: ^10
```